roles/k8s_best_practices_certsuite/templates/certsuite_config.yml.j2

```diff
@@ -50,6 +50,15 @@ servicesignorelist:
   - {{ item }}
 {% endfor %}
 
+{% if kbpc_cwe_apikey is defined and kbpc_cwe_project_id is defined %}
+connectAPIConfig:
```

Comment:
When this setting is defined, it will submit the results?

Reply:
yep, that's the idea